src/common/m_variables_conversion.fpp

```diff
@@ -786,7 +787,17 @@ contains
         real(kind(0d0)), dimension(2) :: Re_K
         real(kind(0d0)) :: rho_K, gamma_K, pi_inf_K, dyn_pres_K
 
-        real(kind(0d0)), dimension(:), allocatable :: nRtmp
+        #:if MFC_CASE_OPTIMIZATION
+#ifndef MFC_SIMULATION
+                real(kind(0d0)), dimension(:), allocatable :: nRtmp
+#else
+                real(kind(0d0)), dimension(nb) :: nRtmp
+        real(kind(0d0)), dimension(nb) :: nRtmp
```

Comment:
It looks like there's a duplicate here

Reply:
Not sure how that happened. Fixed now.

Reply:
more concerning: how did the test suite not kill this? I assume because we do not use `--case-optimization` in the test suite anywhere. I propose we add that.